packages/flutter_tools/lib/src/globals.dart

```diff
@@ -118,6 +118,7 @@ ProcessUtils get processUtils => context.get<ProcessUtils>()!;
 const Platform _kLocalPlatform = LocalPlatform();
 Platform get platform => context.get<Platform>() ?? _kLocalPlatform;
 
+/// Contains some messages produced by Flutter tools.
```

Comment:
This is redundant with the class dartdoc, right?

Reply:
since we require explicit types, they would have a reference to the class definition at the callsite.